File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/Patterns/pattern6.cpp
#include <iostream>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    for(i=0;i<=n/2;i++){
        for(j=n/2;j>=i;j--)
            cout<<"*\t";
        for(j=1;j<2*(i+1);j++)
            cout<<"\t";
        for(j=n/2;j>=i;j--)
            cout<<"*\t";
        cout<<endl;
    }
    for(i=n/2;i>0;i--){
        for(j=n/2+1;j>=i;j--)
            cout<<"*\t";
        for(j=2*i;j>1;j--)
            cout<<"\t";
        for(j=n/2+1;j>=i;j--)
            cout<<"*\t";
        cout<<endl;
    }

    //write your code here
    
}

File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/Patterns/Pattern6.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int n=scn.nextInt();
        for(int i=1;i<=n/2+1;i++)
        {
            for(int j=(n+1)/2;j>=i;j--)
            System.out.print("*\t");
            for(int j=1;j<=i;j++)
            System.out.print("\t");
            for(int j=1;j<=i-1;j++)
            System.out.print("\t");
            for(int j=(n+1)/2;j>=i;j--)
            System.out.print("*\t");
            
            
            System.out.println();
        }
        for(int i=1;i<=n/2;i++)
        {
            for(int j=1;j<=i+1;j++)
            System.out.print("*\t");
            for(int j=i;j<=n/2;j++)
            System.out.print("\t");
            for(int j=n/2;j>=i+1;j--)
            System.out.print("\t");
            for(int j=1;j<=i+1;j++)
            System.out.print("*\t");
            
            System.out.println();
        }
    }
}
    

    

    
